backend/utils/testing/factories/ecommerce.py

```diff
@@ -0,0 +1,33 @@
+import random
+import factory
+from apps.ecommerce.models import Order, Product
+from factory.django import DjangoModelFactory
+from faker import Factory
+import uuid
+
+from utils.testing.factories.organizations import OrganizationFactory
+
+
+fake = Factory.create()
+cars = ["Luna", "Nova", "Atmos", "Eld", "Gnist", "Vilje", "Arctos", "Aquilo", "Borealis"]  # Atmos <3
+
+
+class ProductFactory(DjangoModelFactory):
+    class Meta:
+        model = Product
+
+    name = fake.first_name() + " " + random.choice(cars)
+    price = str(fake.random_int(1, 4000000))
```

Comment:
Ideally we want to avoid randomized test attributes as they can quickly lead to flaky tests, also, `str`?